utils/version.py

```diff
@@ -173,8 +173,9 @@ def parse(self, vstring):
             vstring = ".".join(map(str, vstring))
         elif vstring:
             vstring = str(vstring).strip()
-        if vstring in ('master', 'latest', 'upstream') or 'euwe' in vstring:
+        if vstring in ('master', 'latest', 'upstream') or 'fine' in vstring:
             vstring = 'master'
+        # TODO These aren't used anywhere - remove?
```

Comment:
dunno, this should have been a dictionary on the top of the file since its inception anyway.